ssv/sync_committee_aggregator.go

```diff
@@ -279,17 +271,12 @@ func (r *SyncCommitteeAggregatorRunner) expectedPostConsensusRootsAndDomain() ([
 // 3) Once consensus decides, sign partial contribution data (for each subcommittee) and broadcast
 // 4) collect 2f+1 partial sigs, reconstruct and broadcast valid SignedContributionAndProof (for each subcommittee) sig to the BN
 func (r *SyncCommitteeAggregatorRunner) executeDuty(duty *types.Duty) error {
-	indexes, err := r.GetBeaconNode().GetSyncSubcommitteeIndex(duty.Slot, duty.PubKey)
-	if err != nil {
-		return errors.Wrap(err, "failed fetching sync subcommittee indexes")
-	}
-
 	// sign selection proofs
 	msgs := PartialSignatureMessages{
 		Type:     ContributionProofs,
 		Messages: []*PartialSignatureMessage{},
 	}
-	for _, index := range indexes {
+	for _, index := range r.GetState().StartingDuty.ValidatorSyncCommitteeIndices {
```

Comment:
we need validation for  r.GetState().StartingDuty.ValidatorSyncCommitteeIndices?

Reply:
no need